raylib.ads

```diff
@@ -13,6 +13,12 @@ package Raylib is
             Import => True,
             Convention => C,
             External_Name => "CloseWindow";
+    procedure ToggleFullscreen
```

Comment:
This should be `Toggle_Fullscreen` to be coherent with the conventional Ada style that Tsoding is using.

Reply:
changed